lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -589,6 +597,12 @@ def generate_imsi(self, prefix=None):
         print("Using subscriber IMSI %s" % imsi)
         return imsi
 
+    def update_ipv6_address(self, ue_id, ipv6_addr):
```

Comment:
where is this used?

Reply:
This function will be used in the test script to update ipv6 address to _ue_ip_map like how it is done for ipv4 address during attach. This will later be used to send traffic request to trfserver for data path.